src/test/kotlin/no/nav/familie/dokument/ApplicationLocalSoknadFakedings.kt

```diff
@@ -9,8 +9,8 @@ import org.springframework.boot.autoconfigure.SpringBootApplication
 class ApplicationLocalSoknadFakedings
```

Comment:
Var litt usikker på om jeg skulle gå over til å kalle denne `ApplicationLocalSoknad` nå, men siden sak/brev-launcheren ikke bruker fakedings så kan det på en måte gi mening å beholde det også 🤷‍♂️ jeg er veldig åpen for begge deler, kom gjerne med innspill

Reply:
Kan godt kalle denne for bare "Soknad" - er jo der du har ryddet vekk den gamle :)